packages/core/src/router/metadata.ts

```diff
@@ -167,3 +169,103 @@ export function operation(verb: string, path: string, spec: OperationObject) {
     );
   };
 }
+
+/**
+ * Describe an input parameter of a Controller method.
+ *
+ * @param paramSpec Parameter specification.
+ */
+export function param(paramSpec: ParameterObject) {
```

Comment:
This is exported - but not tested? Seems like we should choose one or the other - exported and tested or not exported and not tested.